parm/post/oceanice_products_sfs.yaml

```
@@ -1,16 +1,17 @@
 ocnicepost:
   executable: "ocnicepost.x"
   namelist:
-    write_grib2: True
-    write_netcdf: False
+    write_grib2: {{ write_grib2 }}
+    write_netcdf: {{ write_netcdf }}
```

```suggestion
    write_netcdf: False
```
Should remain `False` for the reasons above in previous comments.